src/main/java/domain/LottoSystem.java

```diff
@@ -0,0 +1,30 @@
+package domain;
+
+import java.util.List;
+
+public class LottoSystem {
+
+    private final Price price;
+    private final List<LottoTicket> lottoTickets;
+
+    private LottoSystem(final Price price) {
+        this.price = price;
+        this.lottoTickets = LottoMachine.generateLottoTickets(price);
+    }
+
+    public static LottoSystem init(final String price) {
+        return new LottoSystem(Price.valueOf(price));
+    }
+
+    public List<LottoTicket> getLottoTickets() {
+        return lottoTickets;
+    }
```

Comment:
```suggestion
    public List<LottoTicket> getLottoTickets() {
        return Collections.unmodifiableList(lottoTickets);
    }
```
unmodifiableList으로 반환하면 어떤 장점이 있을까요??